spec/models/tag_migration_spec.rb

```diff
@@ -1,6 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe TagMigration do
+  describe '#state' do
```

Comment:
it might be worthwhile wrapping this in a `describe "valid states"` block and having similar assertions for the other valid states. Same with `tagging_spreadsheet_spec.rb`